src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -169,6 +169,22 @@ public class DurableTaskExtension :
         /// </summary>
         public string EventGridKeySettingName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Event Grid publish request retry count.
+        /// </summary>
+        public int EventGridPublishRetryCount { get; set; }

```

Comment:
Do you have any tests which try to assign these new properties?

Reply:
It is difficult to do this test. Is there any good idea?

Reply:
Right now there is a `SetHttpMessageHandler` which allows you to set a preconfigured message handler. Maybe you could change this into a get/set property? Then you could potentially write a simple unit test which sets `EventGridPublishRetryCount`, etc. and checks to see if the `HttpMessageHandler` was correctly configured. Would this work? I apologize if my suggestion is unclear.

Reply:
Thank you. I have added a new test for your review.